cluster/saltbase/salt/flannel-server/network.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Network": "192.168.0.0/16",
```

Comment:
We might want to use this to test very large clusters so please use whatever CIDR block can get you the most nodes...